tests/profiles.spec.ts

```diff
@@ -19,39 +19,36 @@ import {
   visitProfile,
 } from "./helpers/profile";
 
-test("profile create and remove", async ({ page }) => {
-  const profile = randomProfileName();
+let profile = randomProfileName();
+let page: Page;
+test.beforeAll(async ({ browserName, browser }) => {
+  page = await browser.newPage();
+  profile = `${browserName}-${profile}`;
   await createProfile(page, profile);
-  await deleteProfile(page, profile);
 });
 
-test("profile rename", async ({ page }) => {
-  const profile = randomProfileName();
-  await createProfile(page, profile);
+test.afterAll(async () => {
+  await deleteProfile(page, profile);
+  await page.close();
+});
 
+test("profile rename", async () => {
   const newName = profile + "-rename";
   await renameProfile(page, profile, newName);
-
-  await deleteProfile(page, newName);
+  profile = newName;
```

Comment:
Interesting approach, would it work for instances as well? So that we don't have to rename it twice - which is what is currently being done, see lines 48-49 of [instances.spec.ts](https://github.com/canonical/lxd-ui/pull/641/files#diff-53a4f34ae6b1ee87d7500934ca7b7e8daa3c3f826106db78fa47373904db7045)

Reply:
Yeah it would, I think I forgot to apply the same approach to instances after doing it for the other tests.